src/mainboard/clevo/tgl-u/vboot-rwa.fmd

```diff
@@ -6,7 +6,7 @@ FLASH@0xff000000 0x1000000 {
 
 	RW_SECTION_A 0x400000 {
```

Comment:
We need a few adjustments here:
1. Place RW_MISC as first region and define its size (it is fixed so it won't change I believe)
2. Place bootsplash and console as next after RW_MISC
3. Place RW_SECTION_A as last just before WP_RO and do not define the size of RW_SECTION_A, let it stretch to whole free space.

Reply:
Applied the suggestions. Tested, boots correctly.